ios/CodePush/CodePush.h

```diff
@@ -1,5 +1,5 @@
 #if __has_include("RCTEventEmitter.h")
-#import "RCTEventEmitter.h"
+#import <React/RCTEventEmitter.h>
```

Comment:
To make CodePush able to work both with `react-native@0.40.0` and `react-native@0.39.2` we probably should use:
```
#if __has_include("RCTEventEmitter.h")
#import "RCTEventEmitter.h"
#elif __has_include(<React/RCTEventEmitter.h>)
#import <React/RCTEventEmitter.h>
#else
#import "React/RCTEventEmitter.h"   // Required when used as a Pod in a Swift project
#endif
```